indico/htdocs/js/indico/widgets/multiple_items_widget.js

```diff
@@ -202,6 +202,8 @@
                             'text-align': 'center'
                         }
                     };
+                } else if (col.type === 'textarea') {
+                    return {html: item[col.id].replace(/(?:\r\n|\r|\n)/g, '<br>')};
```

Comment:
I think replacing `\n` is enough (the `\r` - even if there is one - is just extra whitespace that gets collapsed, and all systems that used bare `\r` (legacy MacOS) are obsolete/unsupported)

Actually, you could even use `white-space: pre-line` on the container instead of converting anything. This would actually be better because right now any HTML in the content will be evaluated here - not such a good idea considering we don't allow HTML but just linebreaks ;)